src/sentry/conf/types/sdk_config.py

```diff
@@ -3,6 +3,10 @@
 from collections.abc import Callable
 from typing import Any, Literal, NotRequired, TypedDict
 
+# TODO(epurkhiser): This should be imported from the SDK, but it doesn't appear
+# to be exported?
+Event = Any
```

Comment:
just inline `Any` instead of an alias -- aliases are a bad practice most of the time and make it too easy to proliferate hacks like this :)

Reply:
I disagree in this case, I think this makes it clear that these SHOULD be correctly typed, but we're unable to right now due to the missing import. Once the Event import is available we can drop this line and simply import Event from the SDK.

If it was inlined I would have to go through and figure out where to place `Event` again.

Reply:
the problem with this is *someone* is going to get the bright idea to `from sentry.conf.types.sdk_config import Event` 

we already have this problem all over sentry with bad aliases

Reply:
Would it be possible to restrict that?

I really wish python had explicit exports haha

Reply:
could underscore it but that's not going to stop the lowest common denominator :)